packages/tests/src/core/plugins/build-report/index.test.ts

```diff
@@ -0,0 +1,557 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import {
+    serializeBuildReport,
+    unserializeBuildReport,
+} from '@dd/core/plugins/build-report/helpers';
+import type {
+    Input,
+    Entry,
+    File,
+    Options,
+    Output,
+    BuildReport,
+    BundlerReport,
+} from '@dd/core/types';
+import { outputTexts } from '@dd/telemetry-plugins/common/output/text';
+import { defaultDestination, defaultEntry, defaultPluginOptions } from '@dd/tests/helpers/mocks';
+import { BUNDLERS, runBundlers } from '@dd/tests/helpers/runBundlers';
+import path from 'path';
+
+// Used to intercept shared contexts.
+jest.mock('@dd/telemetry-plugins/common/output/text', () => {
+    const originalModule = jest.requireActual('@dd/telemetry-plugins/common/output/text');
+    return {
+        ...originalModule,
+        outputTexts: jest.fn(() => []),
+    };
+});
+
+// Don't send anything.
+jest.mock('@dd/telemetry-plugins/common/sender', () => {
+    const originalModule = jest.requireActual('@dd/telemetry-plugins/common/sender');
+    return {
+        ...originalModule,
+        sendMetrics: jest.fn(() => []),
+    };
+});
+
+const outputTextsMocked = jest.mocked(outputTexts);
+
+const sortFiles = (a: File | Output | Entry, b: File | Output | Entry) => {
+    if (a.name < b.name) {
```

Comment:
using [String.prototype.localeCompare()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) could simplify this maybe?

```ts
// something like:
return a.name.localeCompare(b.name)
```

Reply:
Amazing 🤯